oauth2/src/main/java/com/okta/spring/boot/oauth/ReactiveOktaOAuth2AutoConfig.java

```diff
@@ -69,13 +64,13 @@ OidcReactiveOAuth2UserService oidcUserService(Collection<AuthoritiesProvider> au
     }
 
     @Bean
-    @ConditionalOnBean(ReactiveJwtDecoder.class)
     @ConditionalOnMissingBean(SecurityWebFilterChain.class)
-    SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http, ReactiveJwtDecoder jwtDecoder, ReactiveClientRegistrationRepository clientRegistrationRepository) {
+    SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http, ReactiveJwtDecoder jwtDecoder, ReactiveClientRegistrationRepository clientRegistrationRepository, OktaOAuth2Properties properties) throws Exception {
         // as of Spring Security 5.4 the default chain uses oauth2Login OR a JWT resource server (NOT both)
         // this does the same as both defaults merged together (and provides the previous behavior)
         http.authorizeExchange().anyExchange().authenticated();
         Okta.configureOAuth2WithPkce(http, clientRegistrationRepository);
+        Okta.configureOAuth2WithAudience(http, clientRegistrationRepository, properties);
```

Comment:
This line clobbers the previous lines added parameters. We need to figure out a way to chain them together.

Reply:
@arvindkrishnakumar-okta It seems like we could create the `DefaultServerOAuth2AuthorizationRequestResolver` before passing it to `Okta`'s static methods, but that would change the method signature for `configureOAuth2WithPkce()`. I believe that would be a breaking change and require a major version bump. Do you have any other ideas?

Reply:
@mraible I think that is the only way out here to resolve the clobbering.

Reply:
Would it be possible to overload the method with new signature and deprecate the current one?